shared/p2p/peers_view.go

```diff
@@ -0,0 +1,101 @@
+package p2p
+
+import (
+	"github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"sort"
+)
+
+type PeersView interface {
```

Comment:
Thank you for continuing to diligently scrutinize my exports! :heart: :pray: 

`sortedPeersView` can be unexported as-is. :+1: In retrospect, it wasn't necessary to move the results of refactoring `raintreePeersManager` to `shared/p2p`. I think, at the time, it might have been the simplest way to avoid import loops while I was working on it. 

The `PeersView` interface is referenced by `rainTreePeersManager` which is in the `p2p/raintree` package:
![image](https://user-images.githubusercontent.com/600733/225859604-7cc45e2c-c01a-4dad-8e1d-b9aba07e46ce.png)

I'm still formulating my opinion on whether `peers_view.go` and `peer_manager.go` should be moved into `p2p/types` or `p2p/raintree` but we could avoid exporting the interface in the former case by moving the interface to the consumer package (i.e. `p2p/raintree`). We just wouldn't be able to make the interface assertion side-effect assignment because the interface and implementations will be in different packages and neither would be exported. In the latter case, everything would be in the same package and can be unexported without issue. This path probably also calls for additional refactoring as there already is a `peers_manager.go` in that package directory. Suffice it to say, I've unexported `sortedPeersView`; can we revisit this after #576?

Reply:
`can we revisit this after https://github.com/pokt-network/pocket/pull/576?`

Yup, I just think we need a way to track it for devs reading the code? A comment? A TODO(#XXX)? You have all these thoughts/comments here, but it'll get lost which is what I'mt trying to avoid.

Also, a long time ago someone told me "Treat every day like it's your last day" and it really helps with developing a codebase that minimizes legacy. tl;dr just document your hacks